File: js/taller.js/26.js
let tipoCliente = prompt("Tipo de cliente (1=VIP, 2=Normal):");
let subtotal = parseFloat(prompt("Monto de la compra:"));
let descuentoCliente = 0;

if (tipoCliente === "1") {
    descuentoCliente = subtotal * 0.20;
    console.log("Cliente VIP - 20% aplicado.");
} else if (tipoCliente === "2") {
    descuentoCliente = subtotal * 0.05;
    console.log("Cliente Normal - 5% aplicado.");
} else {
    console.log("Tipo de cliente no válido. Sin descuento.");
}

console.log("Total a pagar: " + (subtotal - descuentoCliente));

File: py/taller.py/26.py
tipo_cliente = input("Tipo de cliente (1=VIP, 2=Normal): ")
subtotal = float(input("Monto de la compra: "))

if tipo_cliente == "1":
    descuento = subtotal * 0.20
    print("Cliente VIP - 20% aplicado.")
elif tipo_cliente == "2":
    descuento = subtotal * 0.05
    print("Cliente Normal - 5% aplicado.")
else:
    descuento = 0
    print("Tipo de cliente no válido. Sin descuento.")

print(f"Total a pagar: {subtotal - descuento}")